internal/pkg/deploy/cloudformation/stack/lb_web_svc.go

```diff
@@ -45,10 +45,10 @@ type LoadBalancedWebService struct {
 	httpsEnabled bool
 
 	// Fields for LoadBalancedWebService that needs a Network Load Balancer.
-	dnsDelegationEnabled   bool // This field is true if the application is associated with a domain. When an ALB is enabled,
-									// `httpsEnabled` has the same value with `dnsDelegationEnabled`, because we enabled https
-									// automatically the app is associated with a domain. When an ALB is disabled, `httpsEnabled`
-									// should always be false; hence they could have different values at this time.
+	dnsDelegationEnabled bool // This field is true if the application is associated with a domain. When an ALB is enabled,
+	// `httpsEnabled` has the same value with `dnsDelegationEnabled`, because we enabled https
+	// automatically the app is associated with a domain. When an ALB is disabled, `httpsEnabled`
+	// should always be false; hence they could have different values at this time.
```

Comment:
nit: this is a bit awkward with the spacing 😂 can we move the comments above the field?

```go
// Fields for LoadBalancedWebService that needs a Network Load Balancer.

//  dnsDelegationEnabled is true if the application is associated with a domain. When an ALB is enabled,
// ....
dnsDelegationEnabled bool
```